docs/fork_update.md

```diff
@@ -0,0 +1,46 @@
+# Updating to OverlayPlugin Fork
+
+OverlayPlugin is moving from <https://github.com/ngld/OverlayPlugin/> to <https://github.com/OverlayPlugin/OverlayPlugin>.
+Unfortunately, nobody has a way to make a release from the old version to update automatically and so you (the user) have to do this manually on your machine (sorry).
+
+The old version will likely continue working for some use cases for some time, but is no longer supported.
```

Comment:
Should this have a note that 6.2 is probably EOL for the old OverlayPlugin and anything beyond that is dragons?

Reply:
Done.  PTAL and tell me if I should word this differently.